soccer/src/soccer/strategy/agent/position/robot_factory_position.hpp

```diff
@@ -69,7 +69,7 @@ class RobotFactoryPosition : public Position {
         current_position_->update_field_dimensions(field_dimensions);
     }
 
-    void update_alive_robots(std::vector<u_int8_t> alive_robots) override {
+    void update_alive_robots(std::array<bool, kNumShells> alive_robots) override {
```

Comment:
```suggestion
    void update_alive_robots(std::array<bool, kNumShells> alive_robots) override {
    Position::update_alive_robots(alive_robots);
```
You need to call super function

Reply:
no, that's not correct. you need to call the super function _for robot factory position_, and then call the current_position function _without Position::_

Reply:
I've updated my GH suggestion to show what I mean